### PR TITLE
feat(text selector): concat sibling text nodes when calculating text

### PR DESCRIPTION
**Pull request by dgozman**
Text that is split into multiple text nodes now matches.

References #1906.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "playwright-core",
-  "version": "0.14.0-post",
+  "version": "1.0.0-post",
```
Comment (pavelfeldman) [inline package-lock.json:3]: Is this change intended?

**Review by pavelfeldman — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (dgozman) [reply package-lock.json:3]: Nothing is intended with package-lock :)
